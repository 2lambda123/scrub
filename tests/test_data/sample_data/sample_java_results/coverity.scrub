coverity001 <Low> :src/testcases/CWE835_Infinite_Loop/CWE835_Infinite_Loop__do_01.java:16: FB.DLS_DEAD_LOCAL_STORE
    /opt/project/test/integration_tests/java_testcase/src/testcases/CWE835_Infinite_Loop/CWE835_Infinite_Loop__do_01.java:16:
    1. defect: Dead store to j.

coverity002 <Low> :src/testcasesupport/AbstractTestCaseClassIssue.java:22: FB.UWF_UNWRITTEN_PUBLIC_OR_PROTECTED_FIELD
    /opt/project/test/integration_tests/java_testcase/src/testcasesupport/AbstractTestCaseClassIssue.java:22:
    1. defect: Unwritten public or protected field: testcasesupport.AbstractTestCaseClassIssue.good1Object.

coverity003 <Low> :src/testcasesupport/AbstractTestCaseClassIssue.java:44: FB.UWF_UNWRITTEN_PUBLIC_OR_PROTECTED_FIELD
    /opt/project/test/integration_tests/java_testcase/src/testcasesupport/AbstractTestCaseClassIssue.java:44:
    1. defect: Unwritten public or protected field: testcasesupport.AbstractTestCaseClassIssue.badObject.

coverity004 <Low> :src/testcasesupport/IO.java:100: FB.DMI_RANDOM_USED_ONLY_ONCE
    /opt/project/test/integration_tests/java_testcase/src/testcasesupport/IO.java:100:
    1. defect: Random object created and used only once.

