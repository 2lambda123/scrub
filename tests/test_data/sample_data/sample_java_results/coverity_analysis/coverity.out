/opt/project/test/integration_tests/java_testcase/src/testcases/CWE835_Infinite_Loop/CWE835_Infinite_Loop__do_01.java:16
  Type: DLS: Dead local store (FB.DLS_DEAD_LOCAL_STORE)

/opt/project/test/integration_tests/java_testcase/src/testcases/CWE835_Infinite_Loop/CWE835_Infinite_Loop__do_01.java:16:
  1. defect: Dead store to j.

/opt/project/test/integration_tests/java_testcase/src/testcasesupport/AbstractTestCaseClassIssue.java:22
  Type: UwF: Unwritten field (FB.UWF_UNWRITTEN_PUBLIC_OR_PROTECTED_FIELD)

/opt/project/test/integration_tests/java_testcase/src/testcasesupport/AbstractTestCaseClassIssue.java:22:
  1. defect: Unwritten public or protected field: testcasesupport.AbstractTestCaseClassIssue.good1Object.

/opt/project/test/integration_tests/java_testcase/src/testcasesupport/AbstractTestCaseClassIssue.java:44
  Type: UwF: Unwritten field (FB.UWF_UNWRITTEN_PUBLIC_OR_PROTECTED_FIELD)

/opt/project/test/integration_tests/java_testcase/src/testcasesupport/AbstractTestCaseClassIssue.java:44:
  1. defect: Unwritten public or protected field: testcasesupport.AbstractTestCaseClassIssue.badObject.

/opt/project/test/integration_tests/java_testcase/src/testcasesupport/IO.java:100
  Type: DMI: Dubious method invocation (FB.DMI_RANDOM_USED_ONLY_ONCE)

/opt/project/test/integration_tests/java_testcase/src/testcasesupport/IO.java:100:
  1. defect: Random object created and used only once.

